assets/app/views/browse/image.html

```diff
@@ -71,10 +71,10 @@
                   <tbody ng-if="(tagsByName | hashSize) == 0">
                     <tr><td colspan="5"><em>{{emptyMessage}}</em></td></tr>
                   </tbody>
-                  <tbody>
-                    <tr ng-repeat-start="tag in tagsByName | orderBy : 'name'">
-                      <td>{{tag.name}}</td>
-                      <td>
+                  <tbody ng-repeat-start="tag in tagsByName | orderBy : 'name'">
```

Comment:
tbody is what does the visual separator at mobile, before we had both <tr> elements in the same <tbody> which was wrong. although i screwed this up, the second tbody needs to be repeated (the history of images for a tag)